src/components/core/core.js

```diff
@@ -184,19 +196,17 @@ export default class Core extends UIObject {
     this.options.mimeType = mimeType
     sources = sources && sources.constructor === Array ? sources : [sources]
     this.containers.forEach((container) => container.destroy())
-    this.mediaControl.container = null
+    this.mediaControl && (this.mediaControl.container = null)
```

Comment:
I think this assignment doesn't make sense anymore since the `MediaControl` now uses the `activeContainer` related events